<ui version="4.0" >
 <class>WizardSubnetRange</class>
 <widget class="QWizardPage" name="WizardSubnetRange" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SubnetRange</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Entrez la plage d'adresse que l'entendue peut distribuer.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Adresse IP de début :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLineEdit" name="lineFirstAddress" >
     <property name="inputMask" >
      <string>000.000.000.000;_</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Adresse IP de fin :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QLineEdit" name="lineLastAddress" >
     <property name="inputMask" >
      <string>000.000.000.000;_</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Un masque de sous réseau définit le nombre de bits d'une adresse IP a utiliser pou rles ID de réseau/sous-réseau, aunsi que le nombre de bits a utiliser pour l'ID de l'hôte? Vous devez specifier le masque de sous-réseau en terme de longueur.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Longueur :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QSpinBox" name="spinLength" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>31</number>
     </property>
     <property name="value" >
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Masque de sous réseau IP :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3" >
    <widget class="QLineEdit" name="lineMask" >
     <property name="text" >
      <string>255.255.255.0</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3" >
    <widget class="QLineEdit" name="lineNetworkAddress" >
     <property name="text" >
      <string>UNKNOWN</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Addresse de réseau :</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QWizardPage</class>
   <extends>QWidget</extends>
   <header>qwizardpage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
